MetaMask/metamask-extension
# feat: new receive flow

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26148?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2383

## **Manual testing steps**

1. Got to the home menu
2. A "Portfolio" button should be on top of the main token display
3. Click on the "Receive" button
4. A QR code should pop up, showing the logo in the middle
5. A new copy address button should be present
6. The address of the account should be displayed (please check video underneath)

## **Screenshots/Recordings**


https://github.com/user-attachments/assets/ca969d32-8852-447d-8be3-62a9cfbae90b



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
